docs/community-toolkit/hosting-rust.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Rust hosting
+author: Alirexaa
+description: Learn how to use the .NET Aspire Rust hosting integration to host Rust applications.
+ms.date: 11/12/2024
+---
+
+# .NET Aspire Rust hosting
+
+[!INCLUDE [includes-hosting](../includes/includes-hosting.md)]
+
+[!INCLUDE [banner](includes/banner.md)]
+
+In this article, you learn how to use the .NET Aspire Rust hosting integration to host Rust applications.
+
+## Hosting integration
+
+To get started with the .NET Aspire Rust hosting integration, install the [📦 CommunityToolkit.Aspire.Hosting.Rust](https://nuget.org/packages/CommunityToolkit.Aspire.Hosting.Rust) NuGet package in the AppHost project.
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package CommunityToolkit.Aspire.Hosting.Rust
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="CommunityToolkit.Aspire.Hosting.Rust"
+                  Version="*" />
+```
+
+---
+
+For more information, see [dotnet add package](/dotnet/core/tools/dotnet-add-package) or [Manage package dependencies in .NET applications](/dotnet/core/tools/dependencies).
+
+## Example usage
```

Comment:
This section should be expanded and the heading renamed to something more meaningful. Also, if there are more APIs or overloads they should be described here as well. [For inspiration, see this example](https://review.learn.microsoft.com/en-us/dotnet/aspire/caching/stackexchange-redis-integration?branch=pr-en-us-2041&tabs=dotnet-cli&pivots=redis#add-redis-resource).